Dockerfile

```diff
@@ -75,7 +75,7 @@ ARG UI_TAG
 ARG UI_RELEASE
 RUN apk add --update --no-cache \
     sqlite=3.44.2-r0 \
-    postgresql16-client=16.3-r0 \
+    postgresql16-client=16.4-r0 \
```

Comment:
Required to get the build working, looks like the old version of the client has been removed. A quick look through the PostgreSQL release notes for this version suggests it just contains bug fixes: https://www.postgresql.org/docs/release/16.4/ 

The extent of testing done on the new client version is the UTs in this repo, if there's any further testing required let me know.